src/mixxx.cpp

```diff
@@ -451,7 +469,15 @@ void MixxxMainWindow::finalize() {
     Timer t("MixxxMainWindow::~finalize");
     t.start();
 
-    // Save the current window state (position, maximized, etc)
+    UserSettingsPointer pConfig = m_pSettingsManager->settings();
+    int inhibit = pConfig->getValue<int>(ConfigKey("[Config]","InhibitScreensaver"));
+    mixxx::ScreenSaverPreference inhiPref = mixxx::ScreenSaverPreference(inhibit);
```

Comment:
This casing form is not easy recognizable as cast. 
Can we switch to 
auto inhiPref = static_cast<mixxx::ScreenSaverPreference>(inhibit);
?